app/controllers/application_controller.rb

```diff
@@ -107,7 +107,7 @@ def api_deprecation_error(exception = nil)
   def smart_proxy_exception(exception = nil)
     process_error(:redirect => :back, :error_msg => exception.message)
     logger.warn "ProxyAPI operation FAILED: #{exception}"
-    logger.debug exception.backtrace.join("\n")
+    logger.warn exception.backtrace.join("\n")
```

Comment:
From earlier comment:

In these instances, we should probably log it as a single entry now they're both at the same log level - so we know they'll always both be logged, they could be one multi-line log message.

If we're always printing backtraces, then we should probably put them through the backtrace silencer - at least, if !debug mode(?).  Put this in a common helper?